### DISCUSSION TITLE
Change the limit?

### DISCUSSION THREAD

**Discussion by bomeara** [r-universe-org/help#251] [General]
I maintain https://phylotastic.r-universe.dev/builds from the repo https://github.com/phylotastic/phylotastic.r-universe.dev. I'm also a maintainer of the CRAN task view for phylogenetics. R-universe is great -- it's helped buffer our community against issues when one core package is at risk of being tossed from CRAN, for example.

However, our universe instance hasn't updated since March. I was trying to debug why it has stopped updating -- turns out as a personal instance it's [limited to 100 packages](https://github.com/r-universe/phylotastic/actions/runs/5068440610/jobs/9100761081), and I currently have 114. 

What's the best procedure here? Is there a way to upgrade to a "community" instance? Should I have one for packages A-M and one for packages N-Z? etc.? I'll temporarily remove 14 packages, but it would be great to have all the ones in our community be available.

And thank you for all the work on R-universe -- it's really great.

**Comment by jeroen** [comment]
OK you seem to have a legitimate use case. I'll bump the global limit to 150.

We have a limit to prevent people from abusing resources by just creating universes with a ton of packages that aren't theirs.

**Reply by jeroen** [reply]
Done via https://github.com/r-universe-org/sync/commit/e38c102e4b959314258d8ee363d479f8b61806af

**Comment by jeroen** [comment]
OK the limit is bumped now, can you fix the remaining problems on https://github.com/r-universe/phylotastic/actions/runs/5068762252/jobs/9101477740 ?

**Comment by bomeara** [comment]
Thanks! I'll fix those now.
